scripts/style-spec.mjs

```diff
@@ -1,184 +1,26 @@
 import referenceSpec from "../scripts/style-spec-reference/v8.json" with { type: "json" };
 
+// https://github.com/maplibre/maplibre-native/issues/2368
```

Comment:
Since you use npm and node, you might consider importing the v8 from the package itself instead of using a copy?

Reply:
Good idea!

Reply:
Done.

Reply:
I'm reverting this. The tests need it too and I don't want to depend on npm to run the tests.